xlml/apis/task.py

```diff
@@ -300,6 +175,19 @@ def clean_up(self, queued_resource: airflow.XComArg) -> DAGNode:
         queued_resource
     )
 
+  with TaskGroup(
```

Comment:
Do you have a test I could take a quick look about graphs?

Reply:
I still need to do more testing. Since this is a big change, I'll will try deploying it to the dev environment and posting links/screenshots.

Reply:
Ran some tests in the dev env and posted screenshots.